configs/MATEKF405WINGV2/config.h

```diff
@@ -0,0 +1,125 @@
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+
+#define BOARD_NAME        MATEKF405WINGV2
+#define MANUFACTURER_ID   MTKS
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_MAX7456
+#define USE_SDCARD
+
+#ifndef USE_WING
+#define USE_WING
+#endif
+
+#ifndef USE_SERVOS
+#define USE_SERVOS
+#endif
+
+#define BEEPER_PIN           PC15
+#define MOTOR1_PIN           PB7
+#define MOTOR2_PIN           PB6
+#define SERVO1_PIN           PB0
+#define SERVO2_PIN           PB1
+#define SERVO3_PIN           PC8
+#define SERVO4_PIN           PC9
+#define SERVO5_PIN           PB14
+#define SERVO6_PIN           PB15
+#define RX_PPM_PIN           PA3
+#define LED_STRIP_PIN        PA15
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PC10
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PC11
+#define UART4_RX_PIN         PA1
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C2_SCL_PIN         PB10
+#define I2C1_SDA_PIN         PB9
+#define I2C2_SDA_PIN         PB11
+#define LED0_PIN             PA14
+#define LED1_PIN             PA13
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PB3
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PC2
+#define SPI3_SDI_PIN         PB4
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PC3
+#define SPI3_SDO_PIN         PB5
+#define ADC_VBAT_PIN         PC0
+#define ADC_RSSI_PIN         PC5
+#define ADC_CURR_PIN         PC1
+#define SDCARD_SPI_CS_PIN    PC14
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+#define USB_DETECT_PIN       PC13
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PB7 , 1,  0) \
+    TIMER_PIN_MAP( 1, PB6 , 1,  0) \
+    TIMER_PIN_MAP( 2, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 3, PB1 , 2,  0) \
+    TIMER_PIN_MAP( 4, PC8 , 2,  1) \
+    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
+    TIMER_PIN_MAP( 6, PB14, 3, -1) \
+    TIMER_PIN_MAP( 7, PB15, 3, -1) \
+    TIMER_PIN_MAP( 8, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 9, PA15, 1,  0) \
+    TIMER_PIN_MAP(10, PA3 , 3, -1) \
+    TIMER_PIN_MAP(11, PA2 , 2,  0)
+
+
+#define SPI3_TX_DMA_OPT     0
+#define ADC1_DMA_OPT        0
+
+
+#define MAG_I2C_INSTANCE I2CDEV_2
+#define BARO_I2C_INSTANCE I2CDEV_1
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_SDCARD
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_OFF
```

Comment:
With dshot bitbang disabled, we lose SPI2 RX (OSD) if two motors are used, but gain gyro DMA. Since SPI_RX is not as important, and a single motor is more likely than a dual motor setup, default bitbang to off. 
```suggestion
#define DEFAULT_DSHOT_BURST DSHOT_DMAR_OFF
#define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_OFF
```

Reply:
There are many modern planes with dual motors, Heewing, Atomrc penguin (I fly it), Swordfish, so this configuration is very likely to be used.
Can you please advice what to do with SPI? 
The draft how to setup betaflight for wings (4.6) is also highly recommended to use dshot.

And it's not clear for me about SPI and OSD.
That config is dumped from my working FC, but I have OSD (MSP displayport), two motors. All is working fine.

Almost all is clear for me, but sorry I'm not a developer and I don't have an idea what to do with dshot, bitbang and spi. 

I need a little help. Thanks in advance.

Reply:
I think it is fine to default to bitbang off. OSD does not really need SPI_RX, and it is not a factor if you have only one motor or are using a digital VTX.